.gitignore

```diff
@@ -122,3 +122,7 @@ env/
 ### vagrant ###
 .vagrant 
 geckodriver.log 
+terraform.tfstate
+.terraform/
+outputs.tfstate
+variables.tf
```

Comment:
Note that:
* We do want to include `variables.tf` as a file, because otherwise Terraform doesn't know what variables to expect, so we don't want to .gitignore it
* Adding this here _doesn't_ remove the file if it's already been staged/committed, so this file has been checked in - along with your secrets!
* Because of the above, we don't want to put any sensitive values into that file

Reply:
Updated, thank you